lib/create-organization-api.ts

```diff
@@ -181,12 +181,15 @@ export default function createOrganizationApi({ http }: { http: AxiosInstance })
      * ```
      */
     getTeams(query: QueryOptions = {}) {
-      return http
-        .get('teams', createRequestConfig({ query }))
-        .then((response) => wrapTeamCollection(http, response.data), errorHandler)
+      const raw = this.toPlainObject() as OrganizationProp
+
+      return endpoints.team
+        .getMany(http, { organizationId: raw.sys.id, query })
```

Comment:
Shouldn't we pass query thought `query: createRequestConfig({ query }).params,` as we do in many other places?

Reply:
Good catch 👍